doc/classes/SceneTree.xml

```diff
@@ -178,10 +178,13 @@
 		<method name="quit">
 			<return type="void">
 			</return>
-			<argument index="0" name="exit_code" type="int" default="-1">
+			<argument index="0" name="exit_code" type="int" default="0">
 			</argument>
 			<description>
-				Quits the application. A process [code]exit_code[/code] can optionally be passed as an argument. If this argument is [code]0[/code] or greater, it will override the [member OS.exit_code] defined before quitting the application.
+				Quits the application. Argument [code]exit_code[/code] can optionally be given (defaulting to 0) to
+				customize the exit status code.
```

Comment:
There shouldn't be manual wrapping, each line == one paragraph.

Reply:
fixed.